### PR TITLE
Change gcse topic tags

### PR DESCRIPTION
**Pull request by Morpheu5**
As per [this](https://trello.com/c/FaGkXZa8/2390-please-enable-gcse-encryption-topic) request.

### CODE REVIEW

File: src/app/services/tagsCS.ts [hunk 1]
```diff
@@ -50,12 +51,12 @@ export class CsTagService extends AbstractBaseTagService {
...
-        {id: TAG_ID.textRepresentation, title: "Representation of text", parent: TAG_ID.dataAndInformation, stageOverride: GCSE_NEW},
-        {id: TAG_ID.imageRepresentation, title: "Representation of images", parent: TAG_ID.dataAndInformation, stageOverride: GCSE_NEW},
+        {id: TAG_ID.numberRepresentation, title: "Representation of numbers", parent: TAG_ID.dataAndInformation, stageOverride: GCSE},
+        {id: TAG_ID.textRepresentation, title: "Representation of text", parent: TAG_ID.dataAndInformation, stageOverride: GCSE},
+        {id: TAG_ID.imageRepresentation, title: "Representation of images", parent: TAG_ID.dataAndInformation, stageOverride: GCSE},
         {id: TAG_ID.soundRepresentation, title: "Representation of sound", parent: TAG_ID.dataAndInformation, stageOverride: GCSE_COMING_2022},
         {id: TAG_ID.compression, title: "Compression", parent: TAG_ID.dataAndInformation, stageOverride: GCSE_COMING_2022},
-        {id: TAG_ID.encryption, title: "Encryption", parent: TAG_ID.dataAndInformation, stageOverride: GCSE_COMING_2022},
+        {id: TAG_ID.encryption, title: "Encryption", parent: TAG_ID.dataAndInformation, stageOverride: GCSE},
```
Comment (jsharkey13) [inline src/app/services/tagsCS.ts:59]: This should be `GCSE_NEW`, no?

File: src/app/services/tagsCS.ts [hunk 2]
```diff
@@ -50,12 +51,12 @@ export class CsTagService extends AbstractBaseTagService {
...
         {id: TAG_ID.fileOrganisation, title: "File organisation", parent: TAG_ID.dataAndInformation, stageOverride: GCSE_HIDDEN},
         {id: TAG_ID.databases, title: "Database concepts", parent: TAG_ID.dataAndInformation, stageOverride: GCSE_COMING_2022},
```
Comment (jsharkey13) [inline src/app/services/tagsCS.ts:61]: There is another request, for databases to also be released (`GCSE_NEW` here).

**Comment by jsharkey13** [comment]
I'm not sure why there are unrelated symbolic question changes in this PR, either?

File: src/app/services/tagsCS.ts (hunk 1 above)
Comment (bh412) [reply src/app/services/tagsCS.ts:59]: Also just to clarify - did Diane specify we want the 'new' to be removed from all the others?

Comment (jsharkey13) [reply src/app/services/tagsCS.ts:59]: Yes, it says as much in the linked Trello card!

**Comment by Morpheu5** [comment]
> I'm not sure why there are unrelated symbolic question changes in this PR, either?

Not sure either. I reverted them.

**Review by jsharkey13 — APPROVED** [review]

**Review by jsharkey13 — APPROVED** [review]

**Pull request merged**